### PR TITLE
Update help.html Link updated  to matrix

### PR DESCRIPTION
**Pull request by prateekiiest**
Link updated  to matrix
@Cadair

### CODE REVIEW

File: help.html [hunk 1]
```diff
@@ -44,9 +44,9 @@ <h1 id="list">The Mailing list</h1>
     <h1 id="chat">Live Chat</h1>
   </div>
   <p>
-  We have a chat room on IRC (#sunpy) hosted on <a href="http://freenode.net">freenode</a> where you can talk 
+  We have a chat room on IRC (#sunpy) hosted on <a href="https://riot.im/app/#/login">freenode</a> where you can talk 
   with SunPy users and developers in real time. If you are unfamiliar with IRC we 
-  recommend you simply use Kiwi IRC's <a href="https://kiwiirc.com/client/irc.freenode.net/#SunPy" target="_blank" >web chat client</a> which will open directly in your browser, or use the embedded client below.
+  recommend you simply use Kiwi IRC's <a href="https://riot.im/app/#/room/#sunpy-general:matrix.org" target="_blank" >web chat client</a> which will open directly in your browser, or use the embedded client below.
```
Comment (Cadair) [inline help.html]: This sentence makes no sense with this link in it! This whole paragraph needs re-wording to be updated for matrix. We should not be pointing users to the IRC channel, we should be directing them to matrix. We could add a line at the end of the paragraph about the matrix channel being bridged into IRC for those who already are using IRC.

Comment (prateekiiest) [inline help.html]: ok, on it

**Comment by prateekiiest** [comment]
Anything remaining to be done here?
@Cadair

File: help.html [hunk 2]
```diff
@@ -46,8 +46,9 @@ <h1 id="chat">Live Chat</h1>
   <p>
   We have a chat room on IRC (#sunpy) hosted on <a href="http://freenode.net">freenode</a> where you can talk
```
Comment (Cadair) [inline help.html]: You are still referencing IRC here, twice!

Also this needs to be in the least technical jargon possible, to make it as accessible for people as possible.

File: help.html [hunk 3]
```diff
@@ -46,8 +46,9 @@ <h1 id="chat">Live Chat</h1>
   <p>
   We have a chat room on IRC (#sunpy) hosted on <a href="http://freenode.net">freenode</a> where you can talk 
   with SunPy users and developers in real time. If you are unfamiliar with IRC we 
-  recommend you simply use Kiwi IRC's <a href="https://kiwiirc.com/client/irc.freenode.net/#SunPy" target="_blank" >web chat client</a> which will open directly in your browser, or use the embedded client below.
+  recommend you simply use matrix.org <a href="https://riot.im/app/#/login" target="_blank" >web chat client</a> which will open directly in your browser, or use the embedded client below.
   Stop by and say hello. Also, most chat programs can connect to the chat room (hostname = irc.freenode.net, channel = #sunpy).
+  The matrix.org channel is bridged into the IRC channel. If you are already using IRC, you can find the link to the matrix.org at the top. You can click there which will redirect you to the below page.
```
Comment (Cadair) [inline help.html]: The motivation of my previous comment was not to get IRC people to switch but to tell them they can join the same channel by being in #sunpy on freenode.

Comment (prateekiiest) [inline help.html]: so, it should be like, 
```
that there are two ways for communication. You can use either of them.
If IRC then above link
else you can check at the matrix.org 
```

File: help.html [hunk 4]
```diff
@@ -44,11 +44,10 @@ <h1 id="list">The Mailing list</h1>
...
   <p>
-  We have a chat room on IRC (#sunpy) hosted on <a href="http://freenode.net">freenode</a> where you can talk 
-  with SunPy users and developers in real time. If you are unfamiliar with IRC we 
-  recommend you simply use matrix.org <a href="https://riot.im/app/#/login" target="_blank" >web chat client</a> which will open directly in your browser, or use the embedded client below.
-  Stop by and say hello. Also, most chat programs can connect to the chat room (hostname = irc.freenode.net, channel = #sunpy).
-  The matrix.org channel is bridged into the IRC channel. If you are already using IRC, you can find the link to the matrix.org at the top. You can click there which will redirect you to the below page.   
+   
+  We have a chat room on <a href="https://riot.im/app/#/room/#sunpy-general:matrix.org" target="_blank" >matrix.org</a> which will open directly in your browser, or use the embedded client below.
+  Stop by and say hello. Talk with the SunPy users and the developers and get started.
+  If you are using IRC hosted on freenode, then you can join the same channel on matrix.org by being in the (#sunpy) channel on freenode.
```
Comment (Cadair) [inline help.html]: much better! I would reword this slightly to make it a little clearer, maybe something like:

"If you are using IRC,  you can join the same channel (bridged into matrix) by joining the (#sunpy) channel on freenode."
